configuration/rules-ng.md

```diff
@@ -5,20 +5,624 @@ title: Next-Gen Rules
 
 {% include base.html %}
 
-# Experimental Next-Gen Rule Engine
+# Next-Generation Rule Engine
 
-Eclipse SmartHome now comes with an initial version of a next-generation rule engine - this is available within openHAB 2 only as an experimental feature.
+[![experimental](https://badges.github.io/stability-badges/dist/experimental.svg)](https://github.com/badges/stability-badges)
 
-> Note: If you use it, this is at your own risk. Especially, you should not expect any documentation besides what is available on the [Eclipse SmartHome website](https://www.eclipse.org/smarthome/documentation/features/rules.html).
+Since openHAB 2.4 another Rule Engine has been added. It works fundamentally different than what you find with our current [Rules](https://www.openhab.org/docs/configuration/rules-dsl.html). It allows Rules to be edited in a graphical fashion and to interact with [JSR223 Scripts (Javascript, Jypthon, etc)](https://www.openhab.org/docs/configuration/jsr223.html).
+
+> Note: If you use it, this is at your own risk. Especially, you should not expect any documentation or community experience besides what you find on this page.
 
 ## Installation
 
 Install the rule engine from Add-ons → Misc → Rule Engine (Experimental).
 
 ![rule engine addon](images/rules_ng_addons.png)
 
-## Usage
-
 When you now refresh your browser, you will see a `Rules` menu appearing in the main menu of the Paper UI.
 
 ![new rule](images/rules_ng_new_rule.png)
+
+## Concept
+
+In general this rule engine aims to support rules defined with syntax similar to:
+
+```
+ON item_id state changed IF item_id.state == desired_value THEN item_id2.state = desired_value2 
+```
+
+Each rule can have some basic information like name, tags, description and three module sections (**triggers, conditions, actions**)
+
+
+- The **'triggers'** section is the trigger (eventing) part. 
+
+
+- The **'conditions'** section lists the conditions which act as a filter for the events - actions of the rule will be executed only if the conditions evaluating the event data are satisfied and return 'true'. In case there are multiple conditions in the 'if' section then all of them must be satisfied - logical AND is used 
```

Comment:
Please break down to one line per sentence.
Vuepress can handle that too in lists.